docs/topics/coroutines-basics.md

```diff
@@ -245,35 +245,39 @@ Done
 
 <!--- TEST -->
 
-## Coroutines ARE light-weight
+## Coroutines are light-weight
 
-Run the following code:
+Coroutines are less resource-intensive than JVM threads. Code that exhausts the
+JVM's available memory when using threads can be expressed using coroutines
+without hitting resource limits. For example, the following code launches
+100000 distinct coroutines that each wait 5 seconds and then print a period
+('.'):
```

Comment:
```suggestion
('.'), all the while consuming very little memory:
```
Or something in this vein. Otherwise, the whole point is obscured.

Reply:
Good point! Added.